src/Algebra/Module/Bundles.agda

```diff
@@ -220,7 +249,20 @@ record Bisemimodule (R-semiring : Semiring r ℓr) (S-semiring : Semiring s ℓs
 
   open LeftSemimodule leftSemimodule public
     using ( +ᴹ-commutativeMonoid; +ᴹ-monoid; +ᴹ-semigroup; +ᴹ-magma; +ᴹ-rawMagma
-          ; +ᴹ-rawMonoid; _≉ᴹ_)
+          ; +ᴹ-rawMonoid; rawLeftSemimodule; _≉ᴹ_)
+
+  open RightSemimodule rightSemimodule public
+    using ( rawRightSemimodule )
+
+  rawBisemimodule : RawBisemimodule R.Carrier S.Carrier m ℓm
+  rawBisemimodule = record
+    { Carrierᴹ = Carrierᴹ
```

Comment:
Is this field not redundant?

Reply:
Yes. My development process is write the type signature, write "name = ?", load file, press C-c C-r in the hole to get a record skeleton, fill in all the holes, delete the redundant fields. Sometimes I forget the last step.